md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -564,11 +504,9 @@ RestWrapper apply(@ModelAttribute("processtemplate")
                     LOGGER.debug("procTemplate.setProcessId= " + procTemplate.getProcessId());
                     procTemplate.setProcessTemplateId(processTemplateForInsert.getProcessTemplateId());
                     LOGGER.debug("procTemplate.setProcessTemplateId= " + procTemplate.getProcessTemplateId());
-                    //            List<PropertiesTemplate> propertiesTemplates = s.selectList("call_procedures.select-missing-property-list-for-template-id", procTemplate);
                     List<PropertiesTemplate> propertiesTemplates = processTemplateDAO.selectMissingPropListForT(procTemplate.getProcessId(), procTemplate.getParentProcessId(), procTemplate.getProcessTemplateId());
                     for (PropertiesTemplate propertyTemplates : propertiesTemplates) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)